cypress/utils/plugins/observability-dashboards/otel-v1-apm-service-map-mappings.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
can we create a separate folder for these json files, instead share the same directory as the source code ?

Reply:
Sure- I don't think there is something in the onboarding so I will create a `/utils/plugins/<plugin>/json/` directory